include/Benchmark/PointsToBenchmark.h

```diff
@@ -0,0 +1,85 @@
+#ifndef POINTS_TO_BENCHMARK_H_
+#define POINTS_TO_BENCHMARK_H_
+#include "Benchmark.h"
+#include "llvm/IR/Metadata.h"
+#include "llvm/IR/DebugInfoMetadata.h"
+
+namespace spatial {
+
+enum Soundness {
+  Sound,
+  Unsound,
+};
+class PointsToBenchmarkRunner : public BenchmarkRunner {
+  Soundness SoundOrUnsound;
+  std::map<llvm::Instruction *, std::vector<std::string>> TCResult;
+  std::map<llvm::BasicBlock *, std::vector<int>> PrecisionResult;
+
+public:
+  PointsToBenchmarkRunner() : BenchmarkRunner("PointsTo"){    
+    SoundOrUnsound = Sound;
+  }
+  std::vector<llvm::Value *> extract(llvm::Instruction *Inst);
+  friend std::ostream &operator<<(std::ostream &OS, const PointsToBenchmarkRunner &B);
+  template <class Ty>
+  void evaluate(llvm::Instruction *Inst, std::set<Ty *>, Ty *);
+  spatial::Soundness CheckSoundness();
+  void evaluatePrecision(llvm::Function &);
+};
+
+template <class Ty>
+void PointsToBenchmarkRunner::evaluate(llvm::Instruction *I, std::set<Ty *> A,
```

Comment:
Can you add some comments here to let the analysis writer know about your expectation wrt the arguments of evaluate? 
You may want to add Doxygen-friendly comments (https://llvm.org/docs/CodingStandards.html#doxygen-use-in-documentation-comments).